pybamm/models/submodels/interface/sei/sei_growth.py

```diff
@@ -134,8 +134,7 @@ def get_coupled_variables(self, variables):
         elif SEI_option == "electron-migration limited":
             # Scott Marquis thesis (eq. 5.94)
             eta_inner = delta_phi - phase_param.U_inner
-            j_sei = phase_param.kappa_inner * eta_inner / L_sei_inner
-
+            j_sei = pybamm.EqualHeaviside(eta_inner, 0) * phase_param.kappa_inner * eta_inner / L_sei_inner
```

Comment:
It's better to use `(eta_inner > 0)` or `(eta_inner < 0)` (which one is it?) instead of `pybamm.EqualHeaviside` (since with `EqualHeaviside` isn't obvious if it's greater or less than 0)

Reply:
Thanks for the suggestion. I've made the change, and @brosaplanella Ferran also suggested this. But, I thought direct logical condition wouldn't work in PyBaMM, as I faced a similar problem in some notebook. Anyway, thanks!